docs_user/adoption-attributes.adoc

```diff
@@ -2,7 +2,7 @@
 :context: assembly
 :build: downstream
 
-ifeval::["{build}" == "upstream"]
+ifeval::["{build}" != "downstream"]
```

Comment:
Can you help me understand why you switched this ifeval to downstream? The names listed in the variables below it are upstream-specific, e.g., we don't use "Wallaby" downstream.